example/lib/example.dart

```diff
@@ -23,7 +23,7 @@ class Person {
 
   Person(this.firstName, this.lastName, this.dateOfBirth,
       {this.middleName, this.lastOrder, List<Order> orders})
-      : this.orders = orders ?? <Order>[];
+      : orders = orders ?? <Order>[];
```

Comment:
hmmm I actually much prefer the `this` here for disambiguation purposes

Reply:
:man_shrugging:

Reply:
Yeah, this confused the heck out of me, too.

I guess if it's the value being assigned it **has** to be for `this` – but it is a bit weird.

Reply:
Ya it is technically not necessary but I always have preferred it....